doc/parallel_numpy.rst

```diff
@@ -23,6 +23,12 @@ worker processes.
   your code can release the GIL, then using ``backend="threading"`` is even
   more efficient.
 
+..
```

Comment:
This is a sphinx-comment see [this](http://www.sphinx-doc.org/en/1.4.9/rest.html#comments) so you do not need the "users may ignore this block", unless you thought that users would read the rst file.

I would move this to the top of the file with an additional comment saying something like "py.test fixture when running doctests"